SpatialGDK/Documentation/content/get-started/introduction.md

```diff
@@ -1,9 +1,9 @@
 <%(TOC)%>
 # Get started: Introduction
 
-Get started with the SpatialOS GDK for Unreal by building the SpatialOS Unreal Engine Fork and following the Multiserver Shooter Tutorial. The tutorial shows you how to deploy a project to the cloud and lets you experience a simple shooter game running across two servers. 
+Get started with the SpatialOS GDK for Unreal by building the SpatialOS Unreal Engine Fork and setting up the GDK Starter Template which you can use as a base for your own project running on SpatialOS.
 
-Once you have completed the tutorial, you are then ready to set up and deploy the Starter Template, which you can use as a base for your own Unreal project running on SpatialOS or, if you have an existing Unreal multiplayer project, you can follow our detailed porting guide to get it onto the GDK and running on SpatialOS.
+After this, you can follow the Multiserver Shooter Tutorial which shows you how to deploy a project to the cloud and demos a simple shooter game running across two servers or, if you have an existing Unreal multiplayer project, you can follow the porting guide to get it onto the GDK and running on SpatialOS.
```

Comment:
This is quite a long sentence - maybe split into two bullet points, as an either/or?